libethereum/Executive.h

```diff
@@ -78,6 +78,11 @@ class StandardTrace
 	DebugOptions m_options;
 };
 
+enum CreationContext: short {
```

Comment:
- Prefer `enum class` to unscoped `enum`
- Brace on separate line
- Better use the full word `Transaction` instead of `Tx`